packages/client/src/tx.ts

```diff
@@ -280,22 +286,19 @@ export const createTxEntry = <
         }),
       )
 
-    const getEstimatedFees = async () => {
-      const encoded = (await getEncodedData()).asBytes()
-      const preLen = encoded.length + 103 // TODO: `103` accounts for the extra aprox length that it's added into the extrinsic once it's signed. In the future we should improve this.
-      const len = preLen + compact.enc(preLen).length
-      const args = toHex(mergeUint8(encoded, u32.enc(len)))
+    const getEstimatedFees = async (from: PolkadotSigner, _hinted?: any) => {
+      const fakeSigner = getPolkadotSigner(
+        from.publicKey,
+        "Sr25519",
+        () => fakeSignature,
+      )
```

Comment:
Shall we extract this to the global scope, like `fakeSignature`?

Reply:
As of right now, the only think that could be further extracted into the global scope is the function that returns the `fakeSignature`, though... 🤔 yeah, I will do that. Thanks!